hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -1571,6 +1571,55 @@
       If enabled, SCM will auto create RATIS factor ONE pipeline.
     </description>
   </property>
+
+  <property>
+    <name>hdds.scm.pipeline.choose.policy.impl</name>
+    <value>org.apache.hadoop.hdds.scm.pipeline.choose.algorithms.RandomPipelineChoosePolicy</value>
+    <tag>SCM, PIPELINE</tag>
+    <description>
+      Sets the policy for choosing a pipeline for a Ratis container. The value should be
+      the full name of class which implements org.apache.hadoop.hdds.scm.PipelineChoosePolicy.
+      The class decides which pipeline will be used to find or allocate Ratis containers. If not set,
+      org.apache.hadoop.hdds.scm.pipeline.choose.algorithms.RandomPipelineChoosePolicy
+      will be used as default value.
+
+      The following values can be used,
+      (1) org.apache.hadoop.hdds.scm.pipeline.choose.algorithms.RandomPipelineChoosePolicy
+      : choose a pipeline randomly.
+      (2) org.apache.hadoop.hdds.scm.pipeline.choose.algorithms.HealthyPipelineChoosePolicy
+      : choose a healthy pipeline randomly.
+      (3) org.apache.hadoop.hdds.scm.pipeline.choose.algorithms.CapacityPipelineChoosePolicy
+      : choose the pipeline with lower utilization from two random pipelines. Note that
+      random choose method will be executed twice in this policy.
+      (4) org.apache.hadoop.hdds.scm.pipeline.choose.algorithms.RoundRobinPipelineChoosePolicy
+      : chooses a pipeline in a round robin fashion. Intended for troubleshooting and testing purposes only.
+    </description>
+  </property>
+
+  <property>
+  <name>hdds.scm.pipeline.choose.policy.impl</name>
```

Comment:
Is it a duplicate?

Reply:
yes, copy-pasta, should be `hdds.scm.ec.pipeline.choose.policy.impl`. but removed due to https://github.com/apache/ozone/pull/6832#discussion_r1646553678